test/run-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cd test1 || exit 1
```

Comment:
For other changes, I understand them - but how would these scripts and directory `test` help?

Generally, tests inside `*_test.go` are much better idea - terraform testing harness catches more issues than executing something manually.